samples/nrf9160/mqtt_simple/src/certificates.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#define CLIENT_PRIVATE_KEY \
```

Comment:
Do we really need to authenticate the client in this simple sample? Wouldn't server certificate verification be enough? I'm not even sure if the default server (mqtt.eclipse.org) allows that.

Reply:
Yes, the server certificate should be enough. I have added the CA cert for "test.mosquitto.org".